src/ServiceControlInstaller.Packaging/ServiceControlInstaller.Packaging.csproj

```diff
@@ -121,7 +121,7 @@
       <AuditLearningTransport Include="$(LearningTransportFolder)*.*" Exclude="@(FilesToExclude->'$(LearningTransportFolder)%(identity)')" />
       <AuditInMemoryPersistence Include="$(InMemoryPersistenceFolder)*.*" Exclude="@(FilesToExclude->'$(InMemoryPersistenceFolder)%(identity)')" />
       <AuditRavenDbPersistence Include="$(RavenDbPersistenceFolder)*.*" Exclude="@(FilesToExclude->'$(RavenDbPersistenceFolder)%(identity)')" />
-      <AuditRavenDb5Persistence Include="$(RavenDb5PersistenceFolder)*.*" Exclude="@(FilesToExclude->'$(RavenDb5PersistenceFolder)%(identity)')" />
+      <AuditRavenDb5Persistence Include="$(RavenDb5PersistenceFolder)\**\*.*" Exclude="@(FilesToExclude->'$(RavenDb5PersistenceFolder)%(identity)')" />
```

Comment:
I think we should not be including subfolders as is, it screws all the logging around checking dependencies' versions. What about we zip the RavenDBServer folder _before_ including it and include the zipped version instead of all the files? We then tweak the Audit instance installer to unzip it as a subfolder at installation time.